app/routes.test.js

```diff
@@ -10,6 +10,12 @@ jest.mock('./logger');
 
 const mockLogoutMethod = jest.fn().mockImplementation(() => Promise.resolve({}));
 
+const mockAuthorisedJwtPayload = JSON.stringify({
+  id: '88421113', name: 'Cool Dude', organisation: 'view',
+});
+
+const fakeAuthorisedCookie = `fakeToken=${mockAuthorisedJwtPayload}`;
```

Comment:
can we call it `mockAuthorisedCookie` please

Reply:
👍